test-clients/src/main/java/com/hedera/services/bdd/suites/consensus/SubmitMessageSuite.java

```diff
@@ -117,6 +119,34 @@ private HapiApiSpec messageSubmissionSimple() {
 				);
 	}
 
+	private HapiApiSpec messageSubmissionSizeChange() {
+		return defaultHapiSpec("messageSubmissionSizeChange")
+				.given(
+						newKeyNamed("submitKey"),
+						createTopic("testTopic")
+								.submitKeyName("submitKey")
+				)
+				.when(
+						cryptoCreate("civilian"),
+						submitMessageTo("testTopic")
+								.message("testmessage")
+								.payingWith("civilian")
+								.hasKnownStatus(SUCCESS),
+						fileUpdate(APP_PROPERTIES)
+								.overridingProps(Map.of("consensus.message.maxBytesAllowed", "20")
+								)
+				)
+				.then(
+						submitMessageTo("testTopic")
+								.message("testmessagetestmessagetestmessagetestmessage")
+								.payingWith("civilian")
+								.hasKnownStatus(MESSAGE_SIZE_TOO_LARGE),
+						fileUpdate(APP_PROPERTIES)
+								.overridingProps(Map.of("consensus.message.maxBytesAllowed", "1024")
```

Comment:
I think it wouldn't it be a good option to save the default property of `consensus.message.maxBytesAllowed` before calling the `fileUpdate` to make the change with new size of 20 and then revert to the default instead of 1024 as this might change in the future thus changing the default `APP_PROPERTIES` for other tests?

Reply:
Normally we should do that. But it's a little tricky to do that in the EET framework as of now.